packages/react-components/src/components/CameraButton.tsx

```diff
@@ -273,6 +279,7 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
         split={props.split ?? props.enableDeviceSelectionMenu}
         aria-roledescription={props.enableDeviceSelectionMenu ? strings.cameraButtonSplitRoleDescription : undefined}
         splitButtonAriaLabel={props.enableDeviceSelectionMenu ? splitButtonAriaString : undefined}
+        splitButtonMenuProps={splitButtonMenuProps}
```

Comment:
if someone was previously passing in `splitButtonMenuProps` this will overwright that, could this instead be:
```suggestion
        splitButtonMenuProps={props.splitButtonMenuProps ?? splitButtonMenuProps}
```

Reply:
done